res/i18n/fr/labels.json

```diff
@@ -203,6 +217,7 @@
 		"low_cut": "Coupure basse",
 		"middle": "Milieu",
 		"presence": "Présence",
+		"show_": "Affichage",
```

Comment:
Here the underscore in "show_" means that the text should be shorened to about 4 symbols.
Is it possible to fix?

Reply:
done